core/app/modules/site-editor/assets/js/module.js

```diff
@@ -146,6 +146,21 @@ export default class SiteEditor {
 					video_url: 'https://www.youtube.com/embed/cQLeirgkguA',
 				},
 			},
+			{
+				type: 'loop',
+				icon: 'eicon-custom',
```

Comment:
please change to the new icon

Reply:
@TzviRabinovitch 
Tested on core: 
![image](https://user-images.githubusercontent.com/94352322/179691124-6ac91f9c-91ac-4b61-bf2f-8d51f6ae035f.png)